docs/introduction/foundations/components/integrations/connected-account.mdx

```diff
@@ -54,15 +54,14 @@ print(integration.get_required_variables())
 ```javascript javascript
 import { Composio } from "composio-core";
 
-const client = new Composio({ apiKey: "COMPOSIO_API_KEY" });
+const client = new Composio();
 
 const entity = await client.getEntity({ id: "default" });
 
 // Integration chosen by the user to connect with
-const integration = await client.integrations.get({ integrationId: "integration_id" });
+const integration = await client.integrations.get({ integrationId: "{{integration_id}}" });
```

Comment:
Specify where we can get the integration id, maybe give the link to FE dashboard?